addon/components/au-file-upload.hbs

```diff
@@ -0,0 +1,43 @@
+<div class="js-upload js-upload-bound" data-upload-allow-drop={{if @dataUploadAllowDrop "" "false"}}>
```

Comment:
I think it makes sense to add `...attributes` here in case more classes or attributes are needed?

Reply:
I have no idea :D

Reply:
It seems more like a remnant from a previous implementation which used a vanilla js library or something? Or something testing related? Anyway, fair enough if you want to keep it like this. It just confused me since I couldn't find a reference to the classes, and the argument isn't used anywhere else either.

Reply:
The js classes look like legacy code from somewhere else. I think they can be removed.

Reply:
I think it's from here: https://overheid.vlaanderen.be/webuniversum/webcomponenten-versie-2#uploadveld

Reply:
ok let's remove it in that case